packages/libsql-client/src/lib/driver/HttpDriver.ts

```diff
@@ -25,11 +25,17 @@ export class HttpDriver implements Driver {
         }
 
         const statements = buildStatements(["BEGIN", ...stmts, "COMMIT"]);
-
-        const response = await fetch(this.url, {
+        const reqParams = {
             method: "POST",
             body: JSON.stringify(statements)
-        });
+        };
+
+        let response;
+        if (typeof fetch === "function") {
+            response = await fetch(this.url, reqParams);
+        } else {
+            response = await crossFetch(this.url, reqParams);
+        }
```

Comment:
It might be more elegant to express it like this instead?

```
const f = typeof fetch === "function" ? fetch : crossFetch
const response = await f(...)
```